lib/msal-common/src/request/EndSessionRequest.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AccountInfo } from "../account/AccountInfo";
+
+export type EndSessionRequest = {
+    account: AccountInfo,
+    postLogoutRedirectUri?: string,
+    authority?: string
```

Comment:
Correlation ID would be a good thing to add.

Reply:
Thoughts on also adding either id_token_hint or extraQueryParameters to support the ask in this issue #996 ?

Reply:
I've added correlationId, we can discuss further changes to logout in a separate PR.